pkg/limayaml/defaults.go

```diff
@@ -59,6 +78,16 @@ func FillDefault(y *LimaYAML, filePath string) {
 	if y.Containerd.User == nil {
 		y.Containerd.User = &[]bool{true}[0]
 	}
+	needsContainerdArchives := (y.Containerd.User != nil && *y.Containerd.User) || (y.Containerd.System != nil && *y.Containerd.System)
```

Comment:
Why check the pointers for `nil`? The previous 6 lines just made sure that they can't be `nil`.

Reply:
removed